JCB_Cinema.Infrastructure/Data/Interfaces/IUnitOfWork.cs

```diff
@@ -6,5 +6,6 @@ public interface IUnitOfWork : IDisposable
     {
         public ITRepository<T> Repository<T>() where T : EntityBase;
         public Task<int> SaveToDatabaseAsync();
+        public CinemaDbContext DbContext();
```

Comment:
Nie mozemy tak. Mamy odwoływać się poprzez Repository<MODEL>()

Reply:
Do usunięcia

Reply:
Fakty, trzeba będzie to poprawić. To mogę to zrobić jutro

Reply:
Ja poprawię resztę jutro, albo w środę